tests/test_view_count.py

```
@@ -0,0 +1,185 @@
+        # 새 게시글 생성
+        new_post = PostCreate(title=f"일관성 테스트 {run + 1}", content="내용")
+        post = service.create_post(new_post)
+        post_id = post.id
+        db.commit()
+
+        def worker():
+            from tests.conftest import TestingSessionLocal
+            thread_db = TestingSessionLocal()
+            try:
+                thread_service = PostService(thread_db)
+                thread_service.increase_view_count(post_id)
```

Ruff (0.11.9)

Function definition does not bind loop variable `post_id`

(B023)